index.html

```diff
@@ -3234,41 +3274,40 @@ <h3 id="applying">
         Each member specifies which object a multi-purpose member can be used
         with.
       </p>
-      <section>
+      <section data-dfn-for="ExternalApplicationResource" data-link-for="ExternalApplicationResource">
```

Comment:
`platform` should be defined in the ExternalApplicationResource section. If this is a shared definition, then we should make a note that two things share the same definition.

Reply:
Maybe you can help me wth that in a follow up